### PR TITLE
Feat: Eloquent

### PR DESCRIPTION
**Pull request by Setanna**

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -17,9 +17,14 @@ class User extends Authenticatable
      * @var array<int, string>
      */
     protected $fillable = [
-        'name',
+        'first_name',
+        'last_name',
+        'address',
```
Comment (MathiasDuus) [inline app/Models/User.php]: Adress not in E/R diagram

Comment (Setanna) [reply app/Models/User.php]: Unused address has been removed from user model

**Review by MathiasDuus — APPROVED** [review]
LGTM 👍

**Comment by Setanna** [comment]
[API - Eloquent](https://trello.com/c/yPbkscem/3-api-eloquent)

**Pull request merged**